generate_db_relationship.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+#ddev-generated
+
+# Args:
+# service name, like 'dbmysql' or 'db'
+# db type, like mariadb:10.4 or postgres:14
+
+export dbservice=$1
+export dbtype=$2
+export dbscheme
+export dbport
+
+case $dbtype in
+  mariadb* | mysql*)
+    dbscheme="mysql"
+    dbport=3306
+    rel="mysql"
+    ;;
+
+  postgres*)
+    dbscheme="pgsql"
+    dbport=5432
+    rel="pgsql"
+    ;;
+  default)
+    printf "no recognized dbtype: '${dbtype}'" && exit 1
+    ;;
+esac
+
+read -r -d '' db_stanza <<DB_EOF
+"database": [
+  {
+    "username": "db",
+    "scheme": "${dbscheme}",
+    "service": "${dbservice}",
+    "fragment": null,
+    "ip": "169.254.118.116",
```

Comment:
Is this an fixed IP in DDEV or just dummy? (same question for the other services below)

Reply:
It also would be reasonable to put hostname here, but not sure if that would break something. Changed this to use 255.255.255.255 for now.